### DISCUSSION TITLE
Double quote completion

### DISCUSSION THREAD

**Discussion by meangrape** [community/community#60498] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I use copilot in vim. Like many people's editors when I type `"` and second quote is opened so when typing `"` I end up with `""`. Copilot almost always wants to complete with its own `"` so I end up with `"{code}""` which I then have to backspace over and delete. I sometimes see this with `)`

**Comment by Akash1134** [comment]
Hi @meangrape ,

Thank you for posting on the Copilot community. 

Copilot might show suggestions are not best in every use-case. In a recent evaluation[ in February 2023](https://github.blog/2023-02-14-github-copilot-now-has-a-better-ai-model-and-new-capabilities/), we found that users accepted on average 35% of all completions shown by GitHub Copilot. We also found that on average more than 46% of developers’ code files were generated by GitHub Copilot, and in certain languages like Java that goes up to 61%. 

However, GitHub Copilot does not write perfect code. It is designed to generate the best code possible given the context it has access to, but it doesn’t test the code it suggests so the code may not always work, or even make sense. GitHub Copilot can only hold a very limited context, so it may not make use of helpful functions defined elsewhere in your project or even in the same file. And it may suggest old or deprecated uses of libraries and languages. When converting comments written in non-English to code, there may be performance disparities when compared to English. For suggested code, certain languages like Python, JavaScript, TypeScript, and Go might perform better compared to other programming languages.

Like any other code, code suggested by GitHub Copilot should be carefully tested, reviewed, and vetted. As the developer, you are always in charge.
